source/blog/2018-05-04-the-emberjs-times-issue-45.md

```diff
@@ -1,71 +1,119 @@
 ---
 title: The Ember.js Times - Issue No. 45
-author: all the people
+author: Kenneth Larsen, Sivakumar Kailasam, Amy Lam, Jen Weber, Jessica Jordan
 tags: Recent Posts, Newsletter, Ember.js Times
 alias : "blog/2018/05/04/the-emberjs-times-issue-44.html"
 responsive: true
 ---
 
 Hey there Emberistas!
 
-...
+Lots of Ember programmers and writers have been busy this week! We have several new data-driven RFCs (📝 Requests for Comments), community efforts, and an outstanding update to Ember inspector to present to you.
 
-Here's a recap of what happened in Emberland these days ✨
+As well as a sincere reminder to eat your veggies! 🥒🥕
 
 ---
 
-## [EMBER](your-main-url-here)
+## [I hope my build never lets me down again in Ember 🐹](https://github.com/emberjs/rfcs/pull/328)
 
-<a class="ember-button ember-button--centered" href="#your-main-url-here">Read more</a>
+Have you ever made a change in the source code of your Ember app and subsequently awaited
+the reload of your app's browser tab, to no avail?
+Seconds, minutes, a coffee later—you might have checked your terminal window—then at last, you find that syntax error in your template
+that led to a failing build, preventing the reload.
+
+This week's RFC proposes the automatic inclusion of **<i>ember-cli-build-notifications</i>**
+into new Ember apps. It's an addon providing system notifications for failing Ember app builds by default.
+
+The RFC proposal also discusses several alternative techniques to improve developer ergonomics regarding
+app builds. If you're curious, check out the [full RFC here](https://github.com/emberjs/rfcs/pull/328)
+and leave your thoughts on it!
+
+<a class="ember-button ember-button--centered" href="https://github.com/emberjs/rfcs/pull/328">Read more</a>
 
 ---
 
 ## [Listen to your Broccoli and it will tell you how to eat it](http://www.oligriffiths.com/broccolijs/)
 
 You might already know that Broccoli takes care of your asset pipeline. But do you know **how** it works? [Oli Griffiths](https://twitter.com/oligriffiths) has made an [amazing tutorial](http://www.oligriffiths.com/broccolijs/) on *the asset pipeline for ambitious applications*. It takes you through the basic setup and guides you through the journey of sass preprocessing, debugging and fingerprints.
 
+---
+
+## [Ember Inspector dark mode](https://github.com/emberjs/ember-inspector)
+
+![Ember Inspector Dark Mode Screenshot](/images/blog/emberjstimes/ember-inspector-dark.png)
 
+The Ember Inspector project is on a roll! v3.1.0 has the new component tree and dark mode. Change your theme from light to dark in Chrome/Firefox dev tools. 🖤
```

Comment:
@rwwagner90 in case you have edits

Reply:
Sorry, it's been a busy week 😬. This is fine with me 👍

Reply:
all good, same! Just wanted you to be aware since I got it in at the last minute 👍